masterlist.yaml

```
@@ -3297,6 +3297,17 @@ plugins:
         itm: 35
         udr: 12
 
+  - name: 'Nec-Character_Clair.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/15581/' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ '**DACompilation_GO ESPs**' ]
+        condition: 'active("DACompilation_GO(_SI)?(_APN)?\.esp")'
+      - <<: *useInstead
+        type: say
+        subs: [ '[Sclerocephalus CM Partner Compilation](https://www.nexusmods.com/oblivion/mods/33739/)' ]
+        condition: 'not active("DACompilation_GO(_SI)?(_APN)?\.esp")'
+
```

And we can add the cleaning data as well.

```suggestion
    clean:
      - crc: 0x8AD45B75
        util: 'TES4Edit v4.0.3h (Hotfix 1)'

```